lib/Runtime/Base/JnDirectFields.h

```diff
@@ -517,6 +517,11 @@ ENTRY(tagPublicLibraryCode)
 ENTRY(winglob)
 ENTRY(platform)
 ENTRY(formatToParts)
+ENTRY(FallbackSymbol)
+
+// This symbol is not part of the regular Symbol API and is only used in rare circumstances in Intl.js for backwards compatibility
+// with the Intl v1 spec. It is visible to the user only using Object.getOwnPropertySymbols(Intl.NumberFormat.call(new Intl.NumberFormat())).
```

Comment:
>It is visible to the user only using Object.getOwnPropertySymbols(Intl.NumberFormat.call(new Intl.NumberFormat())). [](start = 26, length = 115)

is it supposed to be visible to the user this way?

Reply:
Oh. We have never had the correct behavior here so I just tried to emulate what the other engines do. I believe both V8 and SM showed the symbol through GetOwnPropertySymbols (since the symbol is defined on `this`, without being an internal property, I am not sure how it could have ever not been user-visible? Perhaps I am missing something.)

Reply:
Ah, you're correct. I was under the incorrect impression that this was using a private symbol that Dan was alluding to [here](https://github.com/tc39/ecma402/pull/84#issuecomment-282483765). It's weird that the spec chose to expose this publicly.

> I am not sure how it could have ever not been user-visible? 

One way would be for the spec to define the lookup using a WeakMap.

Reply:
The rest of the spec works off of internal slots; if FallbackSymbol were an internal slot, then that should serve the same purpose, no? Regardless, my understanding was that the fallback symbol was put in place because intl v2 broke a specific library that was using Intl in a relatively odd way, but since at least Edge has never gotten this behavior correct, I assume that the fallback behavior is too rare to be worth trying to update at this point.